sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -39,7 +39,6 @@ private[ui] class AllExecutionsPage(parent: SQLTab) extends WebUIPage("") with L
     val running = new mutable.ArrayBuffer[SQLExecutionUIData]()
     val completed = new mutable.ArrayBuffer[SQLExecutionUIData]()
     val failed = new mutable.ArrayBuffer[SQLExecutionUIData]()
-
```

Comment:
Hi, @07ARB . I want to give you a tip. It's always better for the contributor to avoid mixing this kind of cosmetic changes. This kind of cosmetics unneeded changes is only distracting the reviewers. :)

Reply:
@dongjoon-hyun , Thank you, i will remember from now.